src/solvers/sLR.ml

```diff
@@ -500,27 +505,16 @@ module TwoPhased =
   functor (S:EqConstrSys) ->
   functor (HM:Hashtbl.S with type key = S.v) ->
   struct
-    include Make (V) (S) (HM)
-    let narrow = narrow S.Dom.narrow
-    let solve box is iv =
-      let sd = solve (fun _ x y -> S.Dom.widen x (S.Dom.join x y)) is iv in
+    module N = Make0 (V) (SolverBox.NarrowOption) (S) (HM)
```

Comment:
It seems that the only reference to the `narrow` defined in line 8 in this file, that was previously dealing with  "exp.no-narrow", is removed here. So the definition in line 8 may be removed.

Reply:
It is still used with `open SLR`:
```
File "src/solvers/sLRterm.ml", line 23, characters 17-23:
23 |     let narrow = narrow S.Dom.narrow
                      ^^^^^^
Error: Unbound value narrow
File "src/solvers/sLRphased.ml", line 23, characters 17-23:
23 |     let narrow = narrow S.Dom.narrow
                      ^^^^^^
Error: Unbound value narrow
```